lib/MTT/MPI/Install/OMPI.pm

```diff
@@ -78,6 +80,11 @@ sub Install {
     $config->{autogen} = $tmp
         if (defined($tmp));
 
+    $tmp = Value($ini, $section, "ompi_autogen_arguments");
+    $tmp =~ s/\n|\r/ /g;
```

Comment:
honestly, i just copy/pasted the ```ompi_configure_arguments``` regex a few lines above.

```perl
     $tmp = Value($ini, $section, "ompi_configure_arguments");
     $tmp =~ s/\n|\r/ /g;
```

is there anything you want me to change ?